mesonbuild/options.py

```diff
@@ -141,7 +141,7 @@ def __setstate__(self, state: T.Dict[str, T.Any]) -> None:
     def __hash__(self) -> int:
         return self._hash
 
-    def _to_tuple(self) -> T.Tuple[str, str, str, MachineChoice, str]:
+    def _to_tuple(self) -> T.Tuple[str, MachineChoice, str]:
```

Comment:
The commit message mentions `_as_tuple` but the commit changes the annotation of `_to_tuple()`